app/src/main/java/no/usn/mob3000/ui/screens/auth/CreateUserScreen.kt

```diff
@@ -89,6 +92,7 @@ fun CreateUserScreen(
                     onValueChange = { confirmPassword = it },
                     label = { Text(stringResource(R.string.auth_create_user_confirm)) },
                     visualTransformation = PasswordVisualTransformation(),
+                    textStyle = TextStyle(color = MaterialTheme.colorScheme.onBackground),
```

Comment:
[See this comment](https://github.com/frigvid/mob3000-group11/pull/27/files#r1847123849).